department-of-veterans-affairs/caseflow
# Anya/4503 judge name

connects #4503 

Testing:
1. Ensure queue is working pointing to Vacols UAT
2. Ensure the join works correctly even if there are nil values for decass table
3. Ensure queue is working locally using fakes

Output: 

![image](https://user-images.githubusercontent.com/3811786/35746510-037a394c-0815-11e8-9033-127248e5659e.png)


![image](https://user-images.githubusercontent.com/3811786/35746534-199a606c-0815-11e8-9ad3-54553a28ba0a.png)
